### ISSUE TITLE
[componentDiagram] group type is always displayed in brackets in the bottom-left corner of the group

### ISSUE THREAD

**Issue by tex0l** [hikerpig/pintora#207]
Hi !

Thanks again for the great work.

Would it be possible to avoid displaying the group type in component diagrams in the bottom-left corner `[rectangle]`?

```pintora
componentDiagram
    rectangle "my group" {
        [A] as A
        [B] as B
        [C] as C
        [D] as D
        B --> C
        A --> C
        C --> D
    }
```

Which renders as:

![canvas](https://github.com/hikerpig/pintora/assets/7548264/ecfed166-3eb4-48f6-b824-c22299e953ae)

Is there an option I'm missing?

**Comment by hikerpig** [comment]
There is no such option yet but we can figure it out.
What description will best suit your need  in componentDiagram ?
1.  An option to simply hide all group type in bottom-left.  This is easier to implement.
2. A grammar to hide this specific type in this group. Harder but no impossible.

**Comment by hikerpig** [comment]
This has been deployed to the online editor, you can [have a try here](https://pintorajs.vercel.app/demo/live-editor/?code=Y29tcG9uZW50RGlhZ3JhbSUwQSUyMCUyMEBwYXJhbSUyMGhpZGVHcm91cFR5cGUlMjB0cnVlJTBBJTBBJTIwJTIwdGl0bGUlM0ElMjBDb21wb25lbnQlMjBEaWFncmFtJTIwRXhhbXBsZSUwQSUyMCUyMHBhY2thZ2UlMjAlMjJAcGludG9yYS9jb3JlJTIyJTIwJTdCJTBBJTIwJTIwJTIwJTIwJTI4JTI5JTIwR3JhcGhpY3NJUiUwQSUyMCUyMCUyMCUyMCUyOCUyOSUyMElSZW5kZXJlciUwQSUyMCUyMCUyMCUyMCUyOCUyOSUyMElEaWFncmFtJTBBJTIwJTIwJTIwJTIwJTVCRGlhZ3JhbSUyMFJlZ2lzdHJ5JTVEJTIwYXMlMjByZWdpc3RyeSUwQSUyMCUyMCU3RCUwQQ%3D%3D&config=JTdCJTIydGhlbWVDb25maWclMjIlM0ElN0IlMjJ0aGVtZSUyMiUzQSUyMmRlZmF1bHQlMjIlN0QlMkMlMjJzZXF1ZW5jZSUyMiUzQSU3QiU3RCUyQyUyMmVyJTIyJTNBJTdCJTdEJTJDJTIyY29tcG9uZW50JTIyJTNBJTdCJTdEJTJDJTIyYWN0aXZpdHklMjIlM0ElN0IlN0QlMkMlMjJtaW5kbWFwJTIyJTNBJTdCJTdEJTdE).

But official patch is still on the way now.

**Comment by hikerpig** [comment]
This is released in v0.6.4
